src/System.Runtime.Extensions/tests/System/UnloadingAndProcessExitTests.netcoreapp1.1.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+namespace System.Tests
+{
+    public class UnloadingAndProcessExitTests : RemoteExecutorTestBase
+    {
+        [Fact]
+        public void UnloadingEventMustHappenBeforeProcessExitEvent()
+        {
+            string fileName = GetTestFilePath();
+
+            Func<string, int> otherProcess = f =>
+            {
+                Action<int> OnUnloading = i => File.AppendAllText(f, String.Format("u{0}", i));
```

Comment:
Is it guaranteed that this file will not exist beforehand? If the test is run twice, just wondering if the second time it would try to append to the same nonempty file as the first time.